src/data/faq.json

```diff
@@ -50,11 +120,70 @@
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "Here we will inform you about the functional impacts of the end of development work on the CWA.",
-                                "anchor": "ramp_down_effects",
+                                "title": "Which functions remain in the app after the end of development?",
+                                "anchor": "ramp_down_features",
                                 "active": false,
                                 "textblock": [
-                                    ""
+                                    "Even after the end of the development work on the CWA, you can still access your already scanned digital certificates and you can use the contact diary. However, you will not be able to scan any more certificates or to renew existing certificates. You will also no longer receive notifications about the certificates. Additionally, you can still view statistics of the Pandemic Radar; you find a corresponding link on the status screen of the app.",
+                                    "In Release 3.2, functionality has been reduced, but this will not be activated until May 1, 2023. Up to and including April 30, 2023, releases 3.1 and 3.2 of the app will therefore not differ in functionality."
+                                ]
+                            },
+                            {
+                                "title": "Can I still obtain test results/digital test certificates via the app?",
+                                "anchor": "ramp_down_tests",
+                                "active": false,
+                                "textblock": [
+                                    "No, this will also no longer be possible from April 20th 2023, since the connection of the test centers to the CWA infrastructure is no longer available. To retrieve test results and lab reports, please use the app of the respective provider or laboratory."

```

Comment:
"April 20th 2023" =>
"April 20, 2023 onwards"

to keep to standard date format and to correct grammar

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/19e4e13371b4215a468ab318569a73ff429312d4, thanks for the correction!